app/assets/stylesheets/partials/_announcements.scss

```diff
@@ -53,7 +53,7 @@
 
   &__content {
     display: grid;
-    grid-template-columns: 1fr 3.15rem 2.2fr;
+    grid-template-columns: 49% 2% 49%;
```

Comment:
If the 2% in the middle is to guarantee a gap in the middle of two equally-sized halves, I think that it's better practice to write this as a two-column layout (`1fr 1fr` probably) with a set `column-gap`. You'd also have to update the `grid-column` on `announcement__image` to live in column 2 instead of 3, but it looks like it'd otherwise fall into place.

Reply:
The `column-gap` property approach was not used because sadly it is not supported by our friend Internet Explorer:
<img width="1067" alt="image" src="https://user-images.githubusercontent.com/565647/74973995-73e80d00-53f2-11ea-8adf-fb2088573120.png">

Reply:
What a fair-weather friend IE is :(